tgui/packages/tgui/interfaces/ForceEvent.js

```diff
@@ -0,0 +1,147 @@
+import { useBackend, useLocalState } from '../backend';
+import { Box, Stack, Button, Icon, Input, Section, Table } from '../components';
+import { Window } from '../layouts';
+import { flow } from 'common/fp';
+import { filter, sortBy } from 'common/collections';
+
+export const ForceEvent = (props, context) => {
+  return (
+    <Window title="Force Event" width={450} height={450}>
+      <Window.Content scrollable>
+        <Stack fill vertical>
+          <Stack.Item>
+            <EventSearch />
+          </Stack.Item>
+          <Stack.Item>
+            <EventOptionsPanel />
+          </Stack.Item>
+          <Stack.Item grow>
+            <EventContent />
+          </Stack.Item>
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+
+export const EventSearch = (props, context) => {
+  const [searchQuery, setSearchQuery] = useLocalState(
+    context,
+    'searchQuery',
+    ''
+  );
+
+  return (
+    <Section>
+      <Stack>
+        <Stack.Item>
+          <Icon name="search" />
+        </Stack.Item>
+        <Stack.Item grow>
+          <Input
+            autoFocus
+            fluid
+            onInput={(e) => setSearchQuery(e.target.value)}
+            placeholder="Search..."
+            value={searchQuery}
+          />
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
+export const EventOptionsPanel = (props, context) => {
+  const { data } = useBackend(context);
+
+  const [announce, setAnnounce] = useLocalState(context, 'announce', true);
+
+  return (
+    <Stack vertical fill>
```

Comment:
Do you need a stack for a single item?

Reply:
Ah, I have been overzealous. I have changed it.